CONTRIBUTING.md

```diff
@@ -17,8 +17,28 @@ contributing to `kubevela` or build a PoC (Proof of Concept).
 2. Kubernetes version v1.16+ with `~/.kube/config` configured.
 3. ginkgo 1.14.0+ (just for [E2E test](./CONTRIBUTING.md#e2e-test))
 4. golangci-lint 1.31.0+, it will install automatically if you run `make`, you can [install it manually](https://golangci-lint.run/usage/install/#local-installation) if the installation is too slow.
+5. kubebuilder v2.X.X
 
-We also recommend you to learn about KubeVela's [design](https://kubevela.io/docs/concepts) before dive into its code.
+##### kubebuilder install:
+
+linux:
+```
+wget https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.0.0-alpha.1/kubebuilder_2.0.0-alpha.1_linux_amd64.tar.gz
+tar -zxvf  kubebuilder_2.0.0-alpha.1_linux_amd64.tar.gz
+sudo mv kubebuilder_2.0.0-alpha.1_linux_amd64 /usr/local/kubebuilder
```

Comment:
sudo mv kubebuilder_2.0.0-alpha.1_linux_amd64/**bin** /usr/local/kubebuilder

Reply:
k8s control plane bin in kubebuilderv2.0.0 bundle is v1.14.1, but kubevela requires  k8s v1.15+.  
Maybe you can change it to kubebuilder v2.3.1 which is used in KubeVela CI.

Reply:
thanks, i will recommit once.